spec/controllers/account/messages_controller_spec.rb

```diff
@@ -8,17 +8,47 @@
     sign_in sender
   end
 
-  describe 'actions' do
-    let(:create_action) do
+  describe 'POST#create' do
+    let(:valid_create_action) do
       post :create, params: {
```

Comment:
don't use xhr requests in let
if you have the same request multiple times move it to before method